vizro-core/hatch.toml

```diff
@@ -26,7 +26,8 @@ dependencies = [
   "chromedriver-autoinstaller>=0.6.4",
   "toml",
   "pyyaml",
-  "openpyxl"
+  "openpyxl",
+  "jupyter",
```

Comment:
I would pledge to add this, make life so much better

Reply:
Fine by me